packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceRenameLightningComponent.test.ts

```diff
@@ -399,5 +399,19 @@ describe('Force Rename Lightning Component', () => {
       expect(exceptionThrownLwc).to.equal(true);
       expect(exceptionThrownAura).to.equal(true);
     });
+
+  });
+
+  describe('getParentDirectoryOfTestFolder function', () => {
+    it('validates getParentDirectoryOfTestFolder', () => {
+      const folders = ['path', 'to', '__tests__'];
+      const folderPath = folders.join(path.sep);
+
+      const parentFolder = folders.slice(0, -1);
+      const parentFolderPath = parentFolder.join(path.sep);
+
+      const parentDirectory = getParentDirectoryOfTestFolder(folderPath);
+      expect(parentDirectory).to.equal(parentFolderPath);
```

Comment:
@jeffb-sfdc - I ended up with your first suggestion.  I think this is a good balance.  Thanks for the input.